### PR TITLE
Added text at bottom of notification emails explaining how to unsubsc…

### PR DESCRIPTION
**Pull request by jnation3406**
…ribe from them

### CODE REVIEW

File: observation_portal/proposals/templates/proposals/requestfailurelimit.txt [hunk 1]
```diff
@@ -11,3 +11,5 @@ Request url: {{ detail_url }}
 Thank you,
 
 {{ organization_name }}
+
+* You received this message because you opted to have notifications enabled. You can disable notifications globally here: {{ observation_portal_base_url }}/accounts/profile, or individually by proposal here: {{ observation_portal_base_url }}/proposals/{{ request.requestgroup.proposal }}
```
Comment (eheinrich) [inline observation_portal/proposals/templates/proposals/requestfailurelimit.txt:15]: Does the asterisk do any formatting here?

**Review by eheinrich — APPROVED** [review]
Looks good! One thing I would add to the text is something saying that turning off notifications globally won't turn off per-proposal notifications - those will have to be turned off separately. I have a feeling that there might be some confusion with that.

File: observation_portal/proposals/templates/proposals/requestfailurelimit.txt (hunk 1 above)
Comment (jnation3406) [reply observation_portal/proposals/templates/proposals/requestfailurelimit.txt:15]: I don't believe so, just raw text. Do you think I should remove it?

Comment (eheinrich) [reply observation_portal/proposals/templates/proposals/requestfailurelimit.txt:15]: No I think it's good to delineate that message in some way

**Pull request merged**